src/views/demoDetails/PlayerList/index.tsx

```diff
@@ -35,13 +37,34 @@ const useStyles = createStyles((theme) => ({
     display: "inline-block",
     verticalAlign: "top",
   },
+  scoreboardDivider: {
+    border: "none",
+    borderTop: "1px solid #444",
+    margin: "16px",
+  },
+  scoreboardPlayerNameHeader: {
+    paddingLeft: "16px",
+    fontWeight: 800,
+    fontSize: "16pt",
+  },
 }));
 
 export default function PlayerList({ gameSummary }: PlayerListProps) {
   const redPlayers: PlayerSummary[] = [];
   const bluPlayers: PlayerSummary[] = [];
   const others: PlayerSummary[] = [];
 
+  const mainPlayer: PlayerSummary | null =
+    gameSummary.players.find((p) => p.user_id === gameSummary.local_user_id) ??
+    null;
+
+  // Used so we can display the scoreboard for any arbitrary player via user interaction
+  const scoreboardRef: MutableRefObject<ScoreboardTable | null> = useRef(null);
```

Comment:
Using a ref for imperative-style state update is somewhat of an antipattern in my opinion.
I suggest using a more declarative style, like this:
```ts
// Remember only the UserId of the currently selected player
const [selectedPlayerUserId, setSelectedPlayerUserId] = useState(
  // Note: I'm not sure if this is always valid
  gameSummary.local_user_id
);

// This is only recalculated when the selectedPlayerUserId or gameSummary changes
const selectedPlayerScoreboard = useMemo(
  () =>
    gameSummary.players.find(
      (player) => player.user_id === selectedPlayerUserId
    ),
  [selectedPlayerUserId, gameSummary]
);

// Not shown here:
// - Update selectedPlayerUserId when clicking on a player
// - Pass selectedPlayerScoreboard to the Scoreboard element
// - Apply a "selected" style to the player box where the user ID equals selectedPlayerUserId
```